### PR TITLE
Fix issues with YACE configuration

### LINKED ISSUE #324: Issues with LocalStack and local MaC development environment
We've identified a few issues with the YACE and LocalStack configuration which need to be updated to integrate better with the MaC framework.

- YACE is configured to expose port 5003 in docker-compose file but it actually listens on port 5000.
- YACE configuration file did not list `maximum` and `average` statistics which are used in the generic mixin.
- Not clear from the LocalStack README documentation how to get data "into" the Grafana dashboard.

### PR DESCRIPTION
**Pull request by samiwelthomasHO**
Fixes #324

### CODE REVIEW

**Review by georgeowusuHO — APPROVED** [review]

**Review by mahrufiqbalHO — APPROVED** [review]

File: monitoring-as-code/src/metric-types.libsonnet [hunk 1]
```diff
@@ -269,7 +269,7 @@
       },
       customSelectorLabels: {
         deadletterQueueName: 'dimension_QueueName',
-        deadletterQueueType: 'queue_type',
+        deadletterQueueType: 'dimension_queue_type',
```
Comment (samiwelthomasHO) [inline monitoring-as-code/src/metric-types.libsonnet:272]: @michaelpearsonHO keen to get your thoughts on this change. To distinguish between standard and dead-letter queue types, MaC seems to check for QueueName matching or not matching `/.+dlq.+/` and a `queue_type` label either matching or not matching "deadletter".

When YACE exports metrics from AWS (at least from what we've seen from testing against LocalStack) it seems to prefix labels with `tag_` or `dimension_` depending on the Tags assigned to the resource or the dimensions of the metrics themselves. I don't see how YACE would have exported a label called "queue_type" unless it was some kind of custom metric. I don't want to change this if it's been proven to work in a real AWS environment though.

Comment (michaelpearsonHO) [reply monitoring-as-code/src/metric-types.libsonnet:272]: Good spot @samiwelthomasHO. This is a hangover from from when we attempted to add an additional label to the EBSA config but they didn't want to adopt it. Looks like this is still being used in the detail dashboards so we need to refactor this in next Sprint.

Comment (samiwelthomasHO) [reply monitoring-as-code/src/metric-types.libsonnet:272]: Ah okay, for now is it okay to change this to `dimension_queue_type`? In the next sprint we can possibly remove altogether if we're only going to be going on QueueName moving forwards.

Comment (michaelpearsonHO) [reply monitoring-as-code/src/metric-types.libsonnet:272]: Perfect.

**Pull request merged**